intel-sgx/pcs/src/tcb_evaluation_data_numbers.rs

```diff
@@ -0,0 +1,226 @@
+
+use chrono::{DateTime, Utc};
+use crate::{io, Error, Platform, Unverified, VerificationType, Verified};
+use serde::{Deserialize, Deserializer, Serialize};
+use serde_json::value::RawValue;
+use std::marker::PhantomData;
+
+#[cfg(feature = "verify")]
+use {
+    mbedtls::alloc::List as MbedtlsList, mbedtls::x509::certificate::Certificate, mbedtls::Error as MbedError, pkix::oid,
+    pkix::pem::PEM_CERTIFICATE, pkix::x509::GenericCertificate, pkix::FromBer, std::ops::Deref,
+};
+
+/// Implementation of the TcbEvaluationDataNumbers model
+/// https://api.portal.grustedservices.intel.com/content/documentation.html#pcs-tcb-eval-data-numbers-model-v1
+#[allow(dead_code)]
```

Comment:
That's not good. Could you please put `allow(unused)` on the specific fields that are unused instead?